docs/modules/user-guide/partials/proc_stage-2-install-a-devfile-registry-to-a-cluster.adoc

```diff
@@ -0,0 +1,35 @@
+[id="stage-2-install-a-devfile-registry-to-a-cluster_{context}"]
+== Stage 2: Install a Devfile Registry to a cluster
+
+The main goal of this stage (on top of the existing devfile registry install mechanism) is to install the images to the image registry.
```

Comment:
`In this stage, install the devfile registry and the images to the image registry.`

Reply:
`In this stage, install the images to the offline image registry.`